### DISCUSSION TITLE
If there are multiple modifiers for an item, how should they be delimited in the CSV wide/tall templates

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#78] [Q&A] [answered]
The specification doesn't describe how to handle multiple modifiers in the CSV format.
The specification should define what delimiter should be used between the modifiers when multiple are included on a single row otherwise different hospitals/vendors will come up with their own approach.  A pipe "|" character between items seems most consistent with how other fields represent more than one value.

Does that seem reasonable?

Also, it would be good if the documentation defined the semantics of multiple modifiers as being AND as it is on claims i.e. all of the modifiers are being applied to the item and when they are all there, the charges on that row apply.  Right now the documentation is ambiguous: "Include any modifier(s) that may change the standard charge that corresponds to hospital items or services.".  It would be much less useful to have the semantics of multiple modifiers be OR since you can achieve an OR in the CSV format by multiple rows each with one of the modifiers and with the rows otherwise identical.

**Comment by RidenourK** [comment, marked as answer]
@nmayle. Thank you for your question. 

Please see the updated “[CSV Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/CSV)” on this GitHub site for examples of how multiple modifiers can be encoded on the same row.

**Reply by nmaylevc** [reply]
I think it would be **very** useful to include the way to encode multiple modifiers in the documentation of that field in the schema for CSV.

Examples are helpful, but many people will not look to the examples for a specification.

**Reply by RidenourK** [reply]
@nmaylevc. Thank you for your suggestion that CMS detail how to encode modifiers in the CSV documentation.
